From ef6c1d98e0702d6849d6bbbe4d08cfccb033d243 Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Mon, 17 Jul 2017 16:45:19 +0200
Subject: comics: Fix extra ";" leading to a warning during installation

The concatenated mime-types end up with a ...mime-type;;mime-type...
pattern, an empty mime-type, which update-desktop-database doesn't
like.

Error in file "/usr/share/applications/evince.desktop": "" is an invalid MIME type ("" does not contain a subtype)

See https://bugzilla.redhat.com/show_bug.cgi?id=1471474

https://bugzilla.gnome.org/show_bug.cgi?id=785026
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: evince-3.18.2/configure.ac
===================================================================
--- evince-3.18.2.orig/configure.ac
+++ evince-3.18.2/configure.ac
@@ -785,7 +785,7 @@ AC_SUBST(TIFF_MIME_TYPES)
 AC_SUBST(APPDATA_TIFF_MIME_TYPES)
 AM_SUBST_NOTMAKE(APPDATA_TIFF_MIME_TYPES)
 if test "x$enable_comics" = "xyes"; then
-        COMICS_MIME_TYPES="application/x-cbr;application/x-cbz;application/x-cb7;application/x-ext-cbr;application/x-ext-cbz;application/x-ext-cb7;"
+        COMICS_MIME_TYPES="application/x-cbr;application/x-cbz;application/x-cb7;application/x-ext-cbr;application/x-ext-cbz;application/x-ext-cb7"
         APPDATA_COMICS_MIME_TYPES=$(echo "<mimetype>$COMICS_MIME_TYPES</mimetype>" | sed -e 's/;/<\/mimetype>\n    <mimetype>/g')
         if test -z "$EVINCE_MIME_TYPES"; then
            EVINCE_MIME_TYPES="${COMICS_MIME_TYPES}"
